server/sonar-server/src/test/java/org/sonar/server/component/index/ComponentIndexTest.java

```diff
@@ -56,18 +53,11 @@
 
   @Rule
   public UserSessionRule userSession = UserSessionRule.standalone();
-  private PermissionIndexerTester authorizationIndexerTester = new PermissionIndexerTester(es);
+  private ComponentIndex index = new ComponentIndex(es.client(), new AuthorizationTypeSupport(userSession));
 
-  private ComponentIndex index;
-  private ComponentIndexer indexer;
-  private OrganizationDto organization;
-
-  @Before
-  public void setUp() {
-    index = new ComponentIndex(es.client(), userSession);
-    indexer = new ComponentIndexer(db.getDbClient(), es.client());
-    organization = OrganizationTesting.newOrganizationDto();
-  }
+  private ComponentIndexer indexer = new ComponentIndexer(db.getDbClient(), es.client());
+  private PermissionIndexerTester authorizationIndexerTester = new PermissionIndexerTester(es, indexer);
+  private OrganizationDto organization = OrganizationTesting.newOrganizationDto();
```

Comment:
usually we don't use `final` in tests. For consistency that would require to fix all the tests.